{"name": "Test of failed attempt to login with the wrong fields", "status": "passed", "description": "Negative test of the endpoint \"Авторизация пользователя\" POST /api/auth/login.Parameterized test, checks impossibility of authorization when any of the mandatory fields is wrong", "parameters": [{"name": "wrong_field", "value": "'email'"}], "start": 1719670161504, "stop": 1719670161626, "uuid": "0e891dbc-849b-4a0b-8f8d-2e4afff48790", "historyId": "a5a8692c0994207a9a4aafe10c9740d5", "testCaseId": "9ec928b196acec01e210e78e4d76d55b", "fullName": "tests.test_user_login.TestUserLogin#test_login_with_any_wrong_field_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "DESKTOP-4KK18J8"}, {"name": "thread", "value": "600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}